tests/test_providers.py

```diff
@@ -241,3 +282,23 @@ def test_datacite_sync(logger, app, db):
         assert provider.pid.status == PIDStatus.NEW
         assert logger.exception.call_args[0][0] == \
             "Failed to sync status from DataCite"
+
+
+@patch('invenio_pidstore.providers.recordid_v2.base32')
+def test_recordidv2_generate_id_calls_base32_generate(patched_base32, app):
+    original_options = app.config.get('PIDSTORE_RECORDID_OPTIONS')
+    app.config['PIDSTORE_RECORDID_OPTIONS'] = {
+        'length': 8,
```

Comment:
Minor: shouldn't this also be changed to 10 / 5 instead of 8 / 4, since we agreed to make it longer?

Reply:
Here I wanted to test that the configuration is picked up. So I needed to choose values different from the default ones.